docs/source/gettingstarted.rst

```diff
@@ -39,6 +39,11 @@ is to enable the NeuroDebian_ repository, and to install datalad as a system
 package::
 
   sudo apt-get install datalad
+  
+A current version of git-annex (as also provided by the NeuroDebian_ 
+repository) can be installed by typing::
+
+  sudo apt-get install git-annex
```

Comment:
This mixes instruction for installation on Debian, with the general installation instructions. `apt-get install datalad` is completely sufficient. This bug is that you had the impression that `pip install` is a good way to start on Debian -- an impression we should not make (at least in this way). I would probably not start with general instructions special ones afterwards, but reverse this: First: install on Debian, install on ..., and then if we got nothing for your platform -> `pip install`